### PR TITLE
address golint warnings for cmd package

### PR DESCRIPTION
**Pull request by candysmurf**

### CODE REVIEW

File: cmd/snapctl/task.go [hunk 1]
```diff
@@ -49,10 +49,15 @@ var (
 )
 
 const (
+	// K constant is the default count for truncating task hit and miss
```
Comment (danielscottt) [inline cmd/snapctl/task.go]: Not sure these are all necessary.  Maybe something like:

```go
// used to truncate task hit and miss counts
// e.g. 1.4K, 3M, etc
```

File: cmd/snapctl/task.go [hunk 2]
```diff
@@ -48,6 +48,9 @@ var (
 	unionParseFormat = timeParseFormat + " " + dateParseFormat
 )
 
+// Constants used to truncate task hit and miss counts
+// e.g. 1k, 1M, etc. As a side note, we probably don't need
+// anything after the M.
```
Comment (tiffanyfj) [inline cmd/snapctl/task.go]: K should be capitalized :). I'm not sure if @danielscottt really meant to have the side note as part of the comment.

Comment (danielscottt) [inline cmd/snapctl/task.go]: +1 on both statements above.

**Comment by danielscottt** [comment]
LGTM

**Pull request merged**